File: wHereRU_Spring/src/main/java/kosa/hdit5spring/whereru/notice/vo/NoticeVO.java
package kosa.hdit5spring.whereru.notice.vo;

import kosa.hdit5spring.whereru.main.vo.MissingBoardVo;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class NoticeVO {

	private int noticeSeq;
	private String notiSender;
	private String notiReceiver;
	private String notiMessage;
	private String notiTime;
	private String notiType;
	private String msgType;
}

File: wHereRU_Android/app/src/main/java/kosa/hdit5/whereru/util/retrofit/main/vo/NoticeVO.kt
package kosa.hdit5.whereru.util.retrofit.main.vo

data class NoticeVO(

    val noticeSeq: Int? = null,
    val notiSender: String? = null,
    val notiReceiver: String? = null,
    val notiMessage: String? = null,
    var notiTime: String? = null,
    var notiType: String? = null,
    var msgType: String? = null
)
